compose/foundation/foundation/src/uikitMain/kotlin/androidx/compose/foundation/gestures/FlingBehavior.uikit.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2024 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.foundation.gestures
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.platform.LocalDensity
+import androidx.compose.ui.unit.Dp
+
+internal val flingBehaviorVisibilityThresholdDp = Dp(2.0f)
```

Comment:
Why common `0.5f` doesn't work? It doesn't seem like a platform thing that requires expect/actual

Reply:
The default value for threshold is "null". The value of 0.5 was used in Overscroll animations.
These values are too low and animation curve produces a very long "tail" - basically when animation it running, but without any visual outcome because changes are too trivial. Because of that and with multiple scrolls on a scene it's possible to confuse end user with unexpected behaviour of scrolls. The user may thing that scroll action is over, however it still executes in code.
Unlike Android, on iOS animation actions stop faster, when the destination goal is reached.
Here, the minimum value of 2.0f (dp/sec) seems a reasonable speed to complete animation because no more significant changes are expected.

Reply:
Let's write this in the comment ). The explanation is very useful

Reply:
> 2.0f (dp/sec)

It wasn't clear at all

> The value of 0.5 was used in Overscroll animations

It uses 0.5/frame there! Why should we do differently?